packages/web/docs/src/content/gateway/deployment/docker.mdx

```diff
@@ -344,3 +344,45 @@ And finally start it (the config file is in the image and doesn't need to be mou
 ```sh
 docker run -p 4000:4000 hive-gateway-w-add-res supergraph
 ```
+
+## Known Security Vulnerabilities (affected CVE)
+
+Security is important for us, and we are analysing on a regular bases our docker images against
+known CVE database using [`trivy`](https://trivy.dev/latest/).
+
+Because it is based on the official Node image, which itself is based on official stable Debain
+image, our Node base image is affected by some CVE.
+
+None of those CVE are affecting directly Hive Gateway software, since it doesn't uses any of the
+affected components. But keep in mind that even if the gateway itself is not vulnerable, your own
+code can use affected components.
+
+If you need a CVE free docker image, we recommend building your own image minimal image that exactly
```

Comment:
do we have docs on how to build a custom image? if so, please add a link.

Reply:
No we don't have this. I think it's a bit out of scope of our docs, since it's very dependent to the user's needs. For us, we just need a JS runtime, we don't have particular expectations about the environment.